src/js/jsx/Search.jsx

```diff
@@ -86,6 +86,7 @@ define(function (require, exports, module) {
                             searchID={SEARCH_BAR_DIALOG_ID}
                             dismissDialog={this._closeSearchBar}
                             executeOption={this._handleOption}
+                            maxOptions={14}
```

Comment:
Or, just set the default the way you want it for `Search` in `SearchBar` below and omit this entirely. (Future clients of `SearchBar` can change the default as they see fit, but we might as well make the default correct for the first client.)